signer/xpi/xpi.go

```diff
@@ -146,15 +155,26 @@ func New(conf signer.Configuration) (s *XPISigner, err error) {
 		if issuerPrivateKey.N.BitLen() < 2048 {
 			return nil, errors.Errorf("xpi: issuer RSA key must be at least 2048 bits")
 		}
+		if conf.RSACacheConfig.MonitorSampleRate < 5*time.Millisecond {
```

Comment:
a lower limit of 5ms is very low. I'd think anything lower than 5 seconds is already too frequent.